docs/tutorials/04_stream/viz_interaction.py

```diff
@@ -0,0 +1,175 @@
+"""
+====================================
+Streaming FURY with user interaction
+====================================
+
+In this tutorial, we show how to use the FURY Streaming system to
+serve an interactive visualization through a web browser.
+
+You can choose between two differnet encodings: WebRTC or MJPEG.
+WebRTC is a more robust option and can be used to perform
+a live streaming with a low-latency connection for exampel using 
+ngrok. However, to use webRTC you need to install the aiortc library.
+
+.. code-block:: bash
+    pip install aiortc
+
+In addition, if you don't have ffmpeg installed, you need 
```

Comment:
Do you need ffmpeg on Linux too?

Reply:
Yes. But just for WebRTC which is now optional